tests/testthat/test-variables.R

```diff
@@ -46,6 +46,13 @@ test_that("can retrieve categories of CategoricalVariable", {
   expect_setequal(categorical_variable_get_categories(var$.variable), values)
 })
 
+test_that("Queuing invalid category errors", {
+  population <- 10
+  state <- CategoricalVariable$new(c('S', 'I', 'R'), rep('S', population))
+  expect_error(variable$queue_update("X", Bitset$new(1)$insert(1)),
+               '*')
```

Comment:
Not essential formatting

```suggestion
  expect_error(
    variable$queue_update("X", Bitset$new(1)$insert(1)),
    '*'
  )
```